src/main/java/com/j2k/worldbox/item/ModItems.java

```diff
@@ -11,6 +14,10 @@ public class ModItems {
     public static final DeferredRegister<Item> ITEMS =
             DeferredRegister.create(ForgeRegistries.ITEMS, WorldBoxMod.MOD_ID);
 
+    public static final RegistryObject<ModSpawnEggItem> CIVILIAN_SPAWN_EGG = ITEMS.register("civilian_spawn_egg",
+            () -> new ModSpawnEggItem(ModEntityTypes.CIVILIAN, 0xFFC8A8, 0xFF4F00,
```

Comment:
what are these magic numbers?

Reply:
colors

Reply:
colors of what?

Reply:
Hmmm... Maybe for mob spawn egg?... Hmmmm